app/styles/navbar.css

```diff
@@ -83,7 +83,7 @@ nav li a:hover {
   font-size: 0.87rem;
   flex-direction: column;
   justify-content: center;
-  background-color: #fffbfb;
+  background-color: var(--color-off-white);
   box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;

```

Comment:
Can we please include these box shadow colors in the variables file too?

Reply:
Sure Rohan. 
Now I will Write CSS Variables name according to our second approach and update them.